topics:
  joint_encoder: /mcu/state/jointURDF
  imu: /mcu/state/imu
  mocap: /mocap_node/spirit/pose
  terrain_map: /terrain_map
  global_plan: /global_plan
  global_plan_discrete: /global_plan_discrete
  foot_plan_continuous: /foot_plan_continuous
  foot_plan_discrete: /foot_plan_discrete
  local_plan: /local_plan
  body_force:
    joint_torques: /body_force/joint_torques
    toe_forces: /body_force/toe_forces
  # joint_command: /spirit/joint_controller/command
  trajectory: /robot_plan
  state: # These should not change for the remapping to work in some launch files
    ground_truth: /state/ground_truth
    estimate: /state/estimate
    trajectory: /state/trajectory
  control:
    grfs: /control/grfs
    trajectory: /control/trajectory
    joint_command: /control/joint_command
    mode: /control/mode
    leg_override: /control/leg_override
  visualization:
    global_plan: /visualization/global_plan
    local_plan: /visualization/local_plan
    global_plan_grf: /visualization/global_plan_grf
    local_plan_grf: /visualization/local_plan_grf
    global_plan_discrete: /visualization/global_plan_discrete
    foot_plan_discrete: /visualization/foot_plan_discrete
    joint_states:
      estimate: /visualization/joint_states/estimate
      ground_truth: /visualization/joint_states/ground_truth
      trajectory: /visualization/joint_states/trajectory
    foot_0_plan_continuous: /visualization/foot_0_plan_continuous
    foot_1_plan_continuous: /visualization/foot_1_plan_continuous
    foot_2_plan_continuous: /visualization/foot_2_plan_continuous
    foot_3_plan_continuous: /visualization/foot_3_plan_continuous